polyaxon_client/tracking/experiment.py

```diff
@@ -217,11 +217,17 @@ def log_framework(self, framework):
 
     @check_no_op
     def log_artifact(self, file_path):
```

Comment:
I am not sure what I was trying to do here, but this is definitely a wrong implementation, this should be simply:

```python
    @check_no_op
    def log_artifact(self, file_path):
        self.outputs_store.upload_file(file_path)

    @check_no_op
    def log_artifacts(self, dir_path):
        self.outputs_store.upload_dir(dirname=dirname)
```